hypothesis-python/src/hypothesis/ghostwriter.py

```diff
@@ -14,8 +14,37 @@
 # END HEADER
 
 """
-WARNING: this module is under development and should not be used... yet.
-See https://github.com/HypothesisWorks/hypothesis/pull/2344 for progress.
+Writing tests with Hypothesis frees you from the tedium of deciding on and
+writing out specific inputs to test.  Now, the ``hypothesis.ghostwriter``
+module can even write your source code for you!
```

Comment:
this sounds like it's writing the source code for the main code rather than for the tests. Maybe "write your tests for you"?